packages/logging-controller/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@metamask/logging-controller",
+  "version": "1.0.0",
```

Comment:
In order to publish the first version of this package, we will want to start this off at 0.0.0 and then bump this to 1.0.0 as part of a release PR. Changing the version of a package inside of a release PR is how our tooling knows that a new release ought to be issued for that package — if this is already 1.0.0 when that PR is created and doesn't get bumped, then we may run into problems.

Reply:
Oh oops I did 0.0.1 i'll drop it to 0